### ISSUE TITLE
Archlinux: trying to download data.zip from makepkg gives 406 Not Acceptable

### ISSUE THREAD

**Issue by DenSpirit** [GoldenGnu/jeveassets#62]
Hello.

I am trying to build latest jeveassets with data.zip update included (https://aur.archlinux.org/packages/jeveassets).
I've noticed that link to release on https://eve.nikr.net/jeveasset now points to github, but data.zip is still being served from eve.nikr.net. However, when I try to download it using makepkg's automation, it gives me 406 Not Acceptable instead of the file.

If I try downloading with browser it works. I have to package this file with the app because the package resides on root FS.

**Comment by DenSpirit** [comment]
Thank you for the response.
I've tried HTTPS, but without success.
Just now, I've attempted to manually download the file with curl (as Archlinux's makepkg does) and I got the following response:
```<h1>Security incident detected</h1><p>Your request was blocked. Please try again later (or don't).</p>```

It could be that your webserver is blocking some user agents from downloading files.

**Comment by GoldenGnu** [comment]
Yes, that is confirmed. You are being blocked by my web hotels firewall (I don't know why, though).
I can not fix this myself, but, I will contact support, to see if they can work around it.

**Comment by GoldenGnu** [comment]
Okay, talked with my web hotel. It should work now. Let me know if it's fixed...

**Comment by DenSpirit** [comment]
Works on my side, thank you.

Does jEveAssets support serving data files from user folder? That way I could try to allow users to update their data without updating the entire app.

**Comment by GoldenGnu** [comment]
ATM. jEveAssets only support the static data files in the program directory.
That said, jEveAssets own update system, should be able to update the static data files no matter the location, given the user have the sudo password. If that is not working, please, create a bug report for it.

If you want an alternative system for your package, I will be happy to look into it, but, please create a new issue for it...
